api/src/main/java/io/opentelemetry/trace/SpanData.java

```diff
@@ -171,6 +172,7 @@ public static SpanData create(
    * @return the end {@code Timestamp}.
    * @since 0.1.0
    */
+  @Nullable
```

Comment:
When reporting an out of bound Span this should not be null.

Reply:
Actually I had no idea this could be passed as `null` and thus resolved by the `Tracer` ;)